ui/scripts/deploy.sh

```diff
@@ -5,18 +5,22 @@ ENV_FILENAME="/app/.env"
 touch $ENV_FILENAME
 
 # Mapping of issuer-level env variables to app-level
-echo "VITE_API_URL=$ISSUER_API_UI_SERVER_URL" >> $ENV_FILENAME
-echo "VITE_API_PASSWORD=$ISSUER_API_UI_AUTH_PASSWORD" >> $ENV_FILENAME
-echo "VITE_API_USERNAME=$ISSUER_API_UI_AUTH_USER" >> $ENV_FILENAME
+echo "VITE_API_URL=$ISSUER_SERVER_URL" >> $ENV_FILENAME
+echo "VITE_API_PASSWORD=$ISSUER_API_AUTH_PASSWORD" >> $ENV_FILENAME
+echo "VITE_API_USERNAME=$ISSUER_API_AUTH_USER" >> $ENV_FILENAME
+echo "VITE_ISSUER_LOGO=$ISSUER_ISSUER_LOGO" >> $ENV_FILENAME
+echo "VITE_ISSUER_NAME=$ISSUER_ISSUER_NAME" >> $ENV_FILENAME
+
 echo "VITE_BLOCK_EXPLORER_URL=$ISSUER_UI_BLOCK_EXPLORER_URL" >> $ENV_FILENAME
 echo "VITE_BUILD_TAG=$ISSUER_UI_BUILD_TAG" >> $ENV_FILENAME
-echo "VITE_ISSUER_DID=$ISSUER_API_UI_ISSUER_DID" >> $ENV_FILENAME
-echo "VITE_ISSUER_LOGO=$ISSUER_API_UI_ISSUER_LOGO" >> $ENV_FILENAME
-echo "VITE_ISSUER_NAME=$ISSUER_API_UI_ISSUER_NAME" >> $ENV_FILENAME
 echo "VITE_WARNING_MESSAGE=$ISSUER_UI_WARNING_MESSAGE" >> $ENV_FILENAME
 echo "VITE_IPFS_GATEWAY_URL=$ISSUER_UI_IPFS_GATEWAY_URL" >> $ENV_FILENAME
 echo "VITE_SCHEMA_EXPLORER_AND_BUILDER_URL=$ISSUER_UI_SCHEMA_EXPLORER_AND_BUILDER_URL" >> $ENV_FILENAME
 
+# TODO: Remove this envs:
+echo "VITE_ISSUER_DID=VITE_ISSUER_DID" >> $ENV_FILENAME
```

Comment:
Why is this kept?

Reply:
@martinsaporiti

Reply:
that was kept to avoid errors in the UI until did creation was coded. We can remove it now.

Reply:
and please check `VITE_ISSUER_DID` is not mandatory in `env.ts` file.

Reply:
And the `did` prop from the `issuer` in the `Env` type.

Reply:
This is pending from this comment:

> We also need to update `.env-ui.sample`, most data there is outdated.
> Same with `ui/.env.sample` which is empty.